package.json

```diff
@@ -222,6 +222,7 @@
     "node-forge": "^0.9.1",
     "opn": "^5.5.0",
     "oppsy": "^2.0.0",
+    "p-retry": "^4.2.0",
```

Comment:
should this be in the root `package.json`, or in the `x-pack` one?

Reply:
I will add it to the `x-pack`